hipify-clang/CMakeLists.txt

```diff
@@ -1,31 +1,14 @@
 cmake_minimum_required(VERSION 2.8.8)
 project(hipify-clang)
 
-set(BUILD_HIPIFY_CLANG 0 CACHE INTERNAL "")
-
-if (HIPIFY_CLANG_LLVM_DIR)
-    find_package(LLVM PATHS ${HIPIFY_CLANG_LLVM_DIR} REQUIRED NO_DEFAULT_PATH)
-else()
-    message(STATUS "hipify-clang will not be built. To build it please specify absolute path to LLVM 3.8 or higher using HIPIFY_CLANG_LLVM_DIR")
-    return()
-endif()
+find_package(LLVM REQUIRED)
```

Comment:
I believe we should leave for users an opportunity for specifying exact LLVM.

Reply:
I missed your words about CMAKE_PREFIX_PATH for exact LLVM. CMAKE_PREFIX_PATH is ok, but add these words to readme, please.

Reply:
> add these words to readme, please.

This was already done: https://github.com/ROCm-Developer-Tools/HIP/pull/236/commits/6a8dc2dd56949b386fb19bb27fc0b1d617047f24#diff-675eed3cdf4ce542842286f5722ee3bbR24

Perhaps this wording isn't clear enough? We can change it if this is confusing or something - I'm not exactly a great writer ;)

Reply:
So do I, let's leave it as it is written.